packages/playground-examples/copy/zh/sections.json

```diff
@@ -0,0 +1,54 @@
+{
+  "sections": [
+    {
+      "name": "JavaScript",
+      "id": "JavaScript",
+      "subtitle": "了解 TypeScript 如何使用最少的附加语法改善 JavaScript 的日常使用。"
+    },
+    {
+      "name": "TypeScript",
+      "id": "TypeScript",
+      "subtitle": "探索 TypeScript 如何扩展 JavaScript，使其更安全，更便利。"
+    },
+    {
+      "name": "3.7",
+      "id": "3.7",
+      "subtitle": "查看 <a href='https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/'> 版本更新说明。</a>."
```

Comment:
Should we remove spaces around `<a>` tag? Also we can make Chinese full-stop out of the `<a>`. Same for the next item.